components/gantt/timeline/views.md

```diff
@@ -175,13 +174,13 @@ The Gantt Timeline provides four predefined views, which dictate how much time a
 ````
 
 
-## Features
-
-* `SlotWidth` - `double` - the width of each individual slot in pixels.
-
-* `RangeStart` - `DateTime` - determines where the view should start. If not provided, the value is calculated based on the data source. If no data is present `DateTime.Now` is used, and the view shows a single major time slot (a day in `DayView`, month in `MonthView` and so on).
+## Views Parameters
```

Comment:
Usually noun adjectives are used in singular - "View Parameters". Check for broken anchor links if you decide to change this.